mllib/src/test/scala/org/apache/spark/mllib/classification/LogisticRegressionSuite.scala

```diff
@@ -38,7 +38,7 @@ object LogisticRegressionSuite {
     scale: Double,
     nPoints: Int,
     seed: Int): java.util.List[LabeledPoint] = {
-    seqAsJavaList(generateLogisticInput(offset, scale, nPoints, seed))
+    seqAsJavaListConverter(generateLogisticInput(offset, scale, nPoints, seed)).asJava
```

Comment:
Do we need `seqAsJavaListConverter`? `asJava` should returns a Java `List` when the input is a Scala `Seq`.